MangoAPI.Presentation/MangoAPI.Presentation.csproj

```diff
@@ -12,6 +12,7 @@
     </PropertyGroup>
 
     <!-- Exclude the project from coverage report -->
+    <PropertyGroup Condition=" '$(RunConfiguration)' == 'MangoAPI' " />
```

Comment:
What is the motivation for this change?

Reply:
Honestly I don't know how this line appeared, might me added by rider